data/open-data-resources.yaml

```diff
@@ -0,0 +1,116 @@
+- title: 'ePSIplatform’s Cases'
+  image:  'static/img/resources/Image10187.png'
+  who_html: 'ePSIplatform'
+  what_html: 'A collection of examples on Public Sector Information (PSI) Re-use, with the main focus on Europe.'
+  number_of_case_studies: "221"
+  countries_covered:  '31 – Austria, Belgium, Bulgaria, Canada (2), Cyprus, Czech Republic, Denmark, Estonia, Finland, France, Germany, Greece, Ireland, Italy, Latvia, Lithuania, Luxembourg, Malta, Netherlands, New Zealand (1), Norway, Poland, Portugal, Romania, Slovakia, Slovenia, Spain, Sweden, Switzerland, United Kingdom, United States (1)'
+  contact_html: 'Phil Archer, W3C; Martin Alvarez-Espinar, CTIC; Daniel Dietrich, OKFN; Makx Dekkers (according to website)'
+  case_studies_url: "http://www.epsiplatform.eu/list/case?field_countries_tid=47&field_keywords_tid=&field_themes_tid=All&sort_by=created&sort_order=DESC"
+
+
+- title: "Follow the Money "
+  image:  "static/img/resources/Image10195.png"
+  who_html: 'A global network of more than 100 organizations supported by ONE and the Open Knowledge Foundation. Other partners include the Transparency &amp; Accountability Initiative, the World Bank, Development Initiatives, and the Revenue Watch Institute.'
+  what_html: 'The network enables organizations working on “various aspects of the Follow the Money agenda – from resource availability, to spending, to service delivery and results in particular sectors to share information, campaign for the inclusion of open data principles in transparency policies, and identify opportunities for collaboration to drive policy change, including through Follow the Money campaigns in specific countries.”<a href="http://followthemoney.net/open-government-partnership.pdf"><span class="CharOverride-2">1</span></a> The repository has a summary of each initiative including the stage of the project and its categorization.'
```

Comment:
sort of preference thing, but multiline strings with `>` might be more readable here for these long html fields in the yaml